modin/core/storage_formats/base/query_compiler.py

```diff
@@ -2088,6 +2088,17 @@ def apply(self, func, axis, *args, **kwargs):
         axis : {0, 1}
             Target axis to apply the function along.
             0 is for index, 1 is for columns.
+        raw : bool, default: False
+            Whether to pass a high-level DataFrame object (False) or a raw representation
```

Comment:
Is this correct description for the parameter given pandas docs? https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html

Reply:
Corrected "high-level DataFrame" -> "high-level Series"